stade/core/templates/data.html

```diff
@@ -513,10 +513,21 @@ <h1 id="2020">2020</h1></a>
             Download (2MB)</a><br>
             33,126 entries of gold standard lesion diagnoses.
           </td>
-          <td colspan="2">
-            Join the <a href="https://www.kaggle.com/c/siim-isic-melanoma-classification/overview">competition
-            on Kaggle</a>.
+          <td>
+            <a class="track-outbound" href="https://isic-challenge-data.s3.amazonaws.com/2020/ISIC_2020_Test_Dicom.zip">Download DICOM (15.3GB)</a>
```

Comment:
FYI, the filename should be `_DICOM.zip`, since it's an acronym and the `_JPEG.zip` does it too.

I do see that the 2020 training ZIP is `Dicom.zip` though, so :shrug: .

Reply:
I went with consistency. Which do you want?